challenge.js

```diff
@@ -34,14 +34,16 @@
  *	Using the same HTML elements as above...
  *
  *	Suppose A is the body element, B, C, and D are all divs,
- *		and E is a p. 
+ *		and E is a p.
  *
  *	Write a CSS selector that can be used to retrieve E from the DOM.
  *
  *	Answer below.
  */
 
-// ANSWER
+// p, E {
+
+//  }
```

Comment:
`p` is right. But E is just our way of referring to the element within the question - it's not part of the selector.